libs/components/packages/src/schematics/migrations/update-8/update-polyfill/update-polyfill.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { stripIndents } from '@angular-devkit/core/src/utils/literals';
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+
+import { take } from 'rxjs/operators';
+
+import { createTestApp } from '../../../testing/scaffold';
+import { getWorkspace } from '../../../utility/workspace';
+
+import { updatePolyfillSchematic } from './update-polyfill.schematic';
+
+describe('updatePolyfill', () => {
+  const runner = new SchematicTestRunner(
+    '@skyux/packages',
+    require.resolve('../../../../collection.json')
+  );
+
+  it('should create an instance', async () => {
```

Comment:
I'm not sure I understand the name of this test.

Reply:
Renamed the test.